datafusion/core/tests/memory_limit/mod.rs

```diff
@@ -76,8 +76,7 @@ async fn group_by_none() {
     TestCase::new()
         .with_query("select median(request_bytes) from t")
         .with_expected_errors(vec![
-            "Resources exhausted: Failed to allocate additional",
-            "AggregateStream",
+            "Resources exhausted: Resources exhausted with top memory consumers (across reservations) are: AggregateStream"
```

Comment:
Do we need to output 2 times `Resource exhausted`?

Reply:
😆  Good point. Message updated.